ci/Dockerfile.fedora

```diff
@@ -0,0 +1,12 @@
+FROM fedora:latest
+
+MAINTAINER csiddharth@vmware.com
+
+
+RUN dnf -y upgrade
```

Comment:
-q as well please

Reply:
Why do quiet? Wouldn't that make it look like the CI is doing nothing for a long period of time?

Reply:
Anyway, I added the quiet option.